### PR TITLE
New Version: RealVNC.VNCViewer version 7.6.1.50931

### LINKED ISSUE #120739: [Update Request]: RealVNC.VNCViewer 7.6.1  (also "current" version 7.6.0 from winget fails to install)
### What type of update are you requesting?

A new version of an existing package

### Current Package Identifier

RealVNC.VNCViewer

### Package Version

7.6.1

### Please describe the changes you would like to see

MSI Installers:  https://downloads.realvnc.com/download/file/viewer.files/VNC-Viewer-7.6.1-Windows-msi.zip

EXE installer: https://downloads.realvnc.com/download/file/viewer.files/VNC-Viewer-7.6.1-Windows.exe

the "current" version known by winget, 7.6.0, is failing to install, but manually downloading the 7.6.1 zip, extracting it and then installing the x64 msi works ok for me.

```
PS C:\WINDOWS\system32> winget upgrade RealVNC.VNCViewer
Found VNC Viewer [RealVNC.VNCViewer] Version 7.6.0.50657
This application is licensed to you by its owner.
Microsoft is not responsible for, nor does it grant any licenses to, third-party packages.
Downloading https://downloads.realvnc.com/download/file/viewer.files/VNC-Viewer-7.6.0-Windows-msi.zip
  ██████████████████████████████  7.37 MB / 7.37 MB
Successfully verified installer hash
Extracting archive...
Successfully extracted archive
Starting package install...
Installer failed with exit code: 1603
```

image

### PR DESCRIPTION
**Pull request by Trenly**
* Resolves #120739
 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/120771)

### CODE REVIEW

**Comment by Trenly** [comment]
Oop, knew I missed a step

**Comment by stephengillie** [comment]
Manual Validation ran overnight.

**Comment by Trenly** [comment]
@mdanish-kh - Do you happen to be online?

**Comment by mdanish-kh** [comment]
Lemme start my VM up.. :D

**Comment by Trenly** [comment]
> Lemme start my VM up.. :D

You rock!

File: manifests/r/RealVNC/VNCViewer/7.6.1.50931/RealVNC.VNCViewer.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+  Scope: machine
+  InstallerUrl: https://downloads.realvnc.com/download/file/viewer.files/VNC-Viewer-7.6.1-Windows.exe
+  InstallerSha256: B3EAC940277CF028CF45C656A1A568DBF1542A9AEABB27ABE7357C3A0CBD1B5C
+  InstallerSwitches:
+    Silent: /qn REBOOT=ReallySuppress
+    SilentWithProgress: /qn REBOOT=ReallySuppress
+  UpgradeBehavior: install
+  ProductCode: '{D0E24BE9-FC62-4BD4-A0B0-5C5162419E05}'
+  AppsAndFeaturesEntries:
+  - DisplayName: VNC Viewer 7.6.1
```
Comment (mdanish-kh) [inline manifests/r/RealVNC/VNCViewer/7.6.1.50931/RealVNC.VNCViewer.installer.yaml:36]:
```suggestion
  - DisplayName: RealVNC Viewer 7.6.1
```

File: manifests/r/RealVNC/VNCViewer/7.6.1.50931/RealVNC.VNCViewer.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+- Architecture: x86
+  InstallerType: zip
+  NestedInstallerType: wix
+  NestedInstallerFiles:
+  - RelativeFilePath: VNC-Viewer-7.6.1-Windows-en-32bit.msi
+  InstallerUrl: https://downloads.realvnc.com/download/file/viewer.files/VNC-Viewer-7.6.1-Windows-msi.zip
+  InstallerSha256: 625488C08D64B5BCAE23625D2BF0D7AF92BF9BAFB5FB37D07BE7EAE0F389BC2C
+  ProductCode: '{CBEC40FF-2D88-44BD-AF46-CC2B83665A3E}'
+  AppsAndFeaturesEntries:
+  - DisplayName: VNC Viewer 7.6.1
```
Comment (mdanish-kh) [inline manifests/r/RealVNC/VNCViewer/7.6.1.50931/RealVNC.VNCViewer.installer.yaml:59]:
```suggestion
  - DisplayName: RealVNC Viewer 7.6.1
```

File: manifests/r/RealVNC/VNCViewer/7.6.1.50931/RealVNC.VNCViewer.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+- Architecture: x64
+  InstallerType: zip
+  NestedInstallerType: wix
+  NestedInstallerFiles:
+  - RelativeFilePath: VNC-Viewer-7.6.1-Windows-en-64bit.msi
+  InstallerUrl: https://downloads.realvnc.com/download/file/viewer.files/VNC-Viewer-7.6.1-Windows-msi.zip
+  InstallerSha256: 625488C08D64B5BCAE23625D2BF0D7AF92BF9BAFB5FB37D07BE7EAE0F389BC2C
+  ProductCode: '{D0E24BE9-FC62-4BD4-A0B0-5C5162419E05}'
+  AppsAndFeaturesEntries:
+  - DisplayName: VNC Viewer 7.6.1
```
Comment (mdanish-kh) [inline manifests/r/RealVNC/VNCViewer/7.6.1.50931/RealVNC.VNCViewer.installer.yaml:71]:
```suggestion
  - DisplayName: RealVNC Viewer 7.6.1
```

File: manifests/r/RealVNC/VNCViewer/7.6.1.50931/RealVNC.VNCViewer.locale.en-US.yaml [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+
+PackageIdentifier: RealVNC.VNCViewer
+PackageVersion: 7.6.1.50931
+PackageLocale: en-US
+Publisher: RealVNC
+PublisherUrl: https://www.realvnc.com/en
+PublisherSupportUrl: https://www.realvnc.com/en/contact-us
+PrivacyUrl: https://www.realvnc.com/en/legal/#privacy
+Author: RealVNC Ltd
+PackageName: VNC Viewer
```
Comment (mdanish-kh) [inline manifests/r/RealVNC/VNCViewer/7.6.1.50931/RealVNC.VNCViewer.locale.en-US.yaml:12]:
```suggestion
PackageName: RealVNC Viewer
```

File: manifests/r/RealVNC/VNCViewer/7.6.1.50931/RealVNC.VNCViewer.installer.yaml [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+  Scope: machine
+  InstallerUrl: https://downloads.realvnc.com/download/file/viewer.files/VNC-Viewer-7.6.1-Windows.exe
+  InstallerSha256: B3EAC940277CF028CF45C656A1A568DBF1542A9AEABB27ABE7357C3A0CBD1B5C
+  InstallerSwitches:
+    Silent: /qn REBOOT=ReallySuppress
+    SilentWithProgress: /qn REBOOT=ReallySuppress
+  UpgradeBehavior: install
+  ProductCode: '{CBEC40FF-2D88-44BD-AF46-CC2B83665A3E}'
+  AppsAndFeaturesEntries:
+  - DisplayName: VNC Viewer 7.6.1
```
Comment (mdanish-kh) [inline manifests/r/RealVNC/VNCViewer/7.6.1.50931/RealVNC.VNCViewer.installer.yaml:21]:
```suggestion
  - DisplayName: RealVNC Viewer 7.6.1
```

**Review by mdanish-kh — CHANGES_REQUESTED** [review]

**Comment by mdanish-kh** [comment]
Given the status of the pipelines - this would require another https://github.com/microsoft/winget-pkgs/labels/Validation-Completed manual label 😢

File: manifests/r/RealVNC/VNCViewer/7.6.1.50931/RealVNC.VNCViewer.installer.yaml [hunk 6]
```diff
@@ -0,0 +1,75 @@
...
+    UpgradeCode: '{FF5C1C64-EE36-4671-8B8F-4C4944DCC330}'
+    ProductCode: '{D0E24BE9-FC62-4BD4-A0B0-5C5162419E05}'
+    InstallerType: msi
+- Architecture: x86
+  InstallerType: portable
+  InstallerUrl: https://downloads.realvnc.com/download/file/viewer.files/VNC-Viewer-7.6.1-Windows-32bit.exe
+  InstallerSha256: B298E360E22906894DE3FE6F9B003054547DF64B478E4438B67B11767F8E4E48
+  AppsAndFeaturesEntries:
+  - DisplayName: VNC Viewer 7.6.1
```
Comment (mdanish-kh) [inline manifests/r/RealVNC/VNCViewer/7.6.1.50931/RealVNC.VNCViewer.installer.yaml:45]:
```suggestion
  AppsAndFeaturesEntries:
  - DisplayName: RealVNC Viewer 7.6.1
```

File: manifests/r/RealVNC/VNCViewer/7.6.1.50931/RealVNC.VNCViewer.installer.yaml [hunk 7]
```diff
@@ -0,0 +1,75 @@
...
+- Architecture: x64
+  InstallerType: portable
+  InstallerUrl: https://downloads.realvnc.com/download/file/viewer.files/VNC-Viewer-7.6.1-Windows-64bit.exe
+  InstallerSha256: 5253A4971F1BC056ADE0B9F78F07A115A605FC00504DE177E10B1DB5890542FA
```
Comment (mdanish-kh) [inline manifests/r/RealVNC/VNCViewer/7.6.1.50931/RealVNC.VNCViewer.installer.yaml:49]:
```suggestion
  InstallerSha256: 5253A4971F1BC056ADE0B9F78F07A115A605FC00504DE177E10B1DB5890542FA
  AppsAndFeaturesEntries:
    - DisplayName: RealVNC Viewer 7.6.1
```

**Review by mdanish-kh — COMMENTED** [review]
missed a couple

**Review by mdanish-kh — APPROVED** [review]

**Comment by Trenly** [comment]
@stephengillie - Since the last commit was a metadata-only change, shouldn't need a full revalidate?

**Pull request merged**